orchagent/routeorch.cpp

```diff
@@ -893,6 +932,21 @@ void RouteOrch::doTask(Consumer& consumer)
             {
                 removeOverlayNextHops(it_nhg.second, it_nhg.first);
             }
+            else if (it_nhg.first.is_srv6_nexthop())
+            {
+                if(it_nhg.first.getSize() > 1)
+                {
+                    if(m_syncdNextHopGroups[it_nhg.first].ref_count == 0)
+                    {
+                      SWSS_LOG_NOTICE("Remove NHG in doTask");
```

Comment:
No information in the log. Please remove

Reply:
Removed the log